### PR TITLE
add reversible_write_counter in oog constant

### PR DESCRIPTION
**Pull request by DreamWuGit**
refer to https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/746#discussion_r976205762

### CODE REVIEW

File: tests/evm/test_oog_constant.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def test_oog_constant_root(tx: Transaction, bytecode: Bytecode):
                 program_counter=0,
                 stack_pointer=1023,
                 gas_left=2,
-                reversible_write_counter=2,
+                reversible_write_counter=0,
```
Comment (han0110) [inline tests/evm/test_oog_constant.py:63]: Why not update the next step to have `rw_counter=28`? I don't think any step will have `reversible_write_counter=0`

**Review by han0110 — APPROVED** [review]
LGTM!

File: tests/evm/test_oog_constant.py (hunk 1 above)
Comment (DreamWuGit) [reply tests/evm/test_oog_constant.py]: sound good & updated in [4879fb5](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/280/commits/4879fb507b64b8da492d85b52508114196e5641d) thanks!

**Review by ed255 — APPROVED** [review]
LGTM!

**Pull request merged**